contrib/create-waku/package.json

```diff
@@ -1,5 +1,15 @@
 {
   "name": "create-waku",
   "version": "0.4.10",
-  "bin": "./cli.js"
+  "bin": "./cli.js",
+  "type": "module",
+  "scripts": {
+    "build": "./node_modules/.bin/esbuild cli.js --bundle --platform=node --outfile=out.cjs",
```

Comment:
wait, you want to build it before publish? we don't have CD yet.

Reply:
Remove esbuild